src/modules/Elsa.Workflows.Core/Contracts/IVariablePersistenceManager.cs

```diff
@@ -6,9 +6,12 @@ namespace Elsa.Workflows;
 public interface IVariablePersistenceManager
 {
     /// <summary>
-    /// Loads the variables into the specified <see cref="WorkflowExecutionContext"/>. 
+    /// Loads the variables into the specified <see cref="WorkflowExecutionContext"/>.
     /// </summary>
-    Task LoadVariablesAsync(WorkflowExecutionContext context);
+    /// <param name="context"></param>
+    /// <param name="excludeValueRetrievalForLargeDataDrivers"></param>
+    /// <returns></returns>
+    Task LoadVariablesAsync(WorkflowExecutionContext context, bool excludeValueRetrievalForLargeDataDrivers = false);
```

Comment:
It would be better to accept a set of Storage Driver Tags, because the term "exclude value retrieval for large data drivers" sounds very specific to a single tag. Given the introduction of tags, which I like, it would follow that we can use it here to filter by.